fabric-resource-conditions-api-v1/src/testmod/resources/resourcepacks/overlay_test/pack.mcmeta

```diff
@@ -0,0 +1,27 @@
+{
+  "pack": {
+    "pack_format": 26,
+    "description": "Fabric Overlay Conditions Test Datapack."
+  },
+  "fabric:overlays": {
```

Comment:
Could we add the conditions to the vanilla `overlays`? If not whats the reasoning?

Reply:
Vanilla overlays parse entries with a codec and Fabric overlays don't, so either resource conditions would need to be refactored to use codecs or there'd have to be mixin hackery to the vanilla overlays to parse both vanilla and fabric overlays at once. Separating the overlays was the cleanest implementation that I saw.

Reply:
It should be easy enough to parse JSON with a codec

Reply:
Wouldn't that require rewriting how resource conditions are created to parse with a Codec rather than with grabbing fields from a JsonObject manually?